scripts/data/jellyfin-web-urls.json

```
@@ -1,4 +1,5 @@
 [
+  "/docs/general/administration/backup-and-restore.html",
```

```suggestion
  "/docs/general/administration/backup-and-restore",
```

The `.html` extension is a legacy thing. It isn't needed on new links since they don't include it.